master/reference/calicoctl/resources/globalnetworkpolicy.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Global Network Policy Resource (GlobalNetworkPolicy)
+---
+
+A Global Network Policy resource (GlobalNetworkPolicy) represents an ordered set of rules which are applied
+to a collection of endpoints which match a [label selector](#selector).
+
+GlobalNetworkPolicy is not a namespaced resource, which means, GlobalNetworkPolicy applies to [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint) in all namespaces. 
```

Comment:
I think you could lose the "which means" and just make these into separate sentences. 
"resource, which means, GlobalNetworkPolicy" --> "resource. GlobalNetworkPolicy"